site/content/docs/5.0/components/carousel.md

```diff
@@ -364,6 +364,10 @@ var carousel = new bootstrap.Carousel(myCarousel, {
     </tr>
     <tr>
       <td><code>nextWhenVisible</code></td>
+      <td>Don't cycle carousel to next if it's not available. <strong>Returns to the caller before the target item has been shown</strong>
```

Comment:
we should mention it use the visible API here but not the `next` method

Reply:
Updated to -->
Don't cycle carousel to next when the page isn't visible or the carousel or its parent isn't visible. Returns to the caller before the target item has been shown